core/src/main/java/org/apache/accumulo/core/fate/user/UserFateStore.java

```diff
@@ -256,21 +256,24 @@ protected Stream<FateIdStatus> getTransactions(EnumSet<TStatus> statuses) {
       RowFateStatusFilter.configureScanner(scanner, statuses);
       TxColumnFamily.STATUS_COLUMN.fetch(scanner);
       TxColumnFamily.RESERVATION_COLUMN.fetch(scanner);
+      TxInfoColumnFamily.TX_NAME_COLUMN.fetch(scanner);
       return scanner.stream().onClose(scanner::close).map(e -> {
         String txUUIDStr = e.getKey().getRow().toString();
         FateId fateId = FateId.from(fateInstanceType, txUUIDStr);
         SortedMap<Key,Value> rowMap;
         TStatus status = TStatus.UNKNOWN;
         FateReservation reservation = null;
+        Fate.FateOperation fateOp = null;
```

Comment:
My impl includes rename of `txName` to `fateOp` in the places where I write new code. Ignore the discrepancy in naming: these are the same and will be consolidated in a later PR via https://github.com/apache/accumulo/issues/5230.